src/main/java/frc/team2412/robot/util/InterpolatingTreeMap.java

```diff
@@ -23,6 +34,60 @@ public InterpolatingTreeMap(ShooterDataDistancePoint[] dataPoints) {
         distanceBiasEntry = ShooterConstants.tab.add("Distance Bias", 0.0).withPosition(0, 0).withSize(1, 1).getEntry();
     }
 
+    /**
+     * Creates a {@link InterpolatingTreeMap} from a path to a CSV file.
+     *
+     * @param fileName
+     *            The path to the CSV file.
+     * @return An {@link InterpolatingTreeMap} from the data in the CSV file.
+     */
+    public static InterpolatingTreeMap fromCsv(String fileName) {
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get(fileName))) {
+            String line;
+            InterpolatingTreeMap map = new InterpolatingTreeMap();
+
+            while ((line = reader.readLine()) != null) {
```

Comment:
It would be good to add a log message for the file name being read.  Also gives context if any lines are skipped.